### PR TITLE
Update Miner Binaries

### PR DESCRIPTION
**Pull request by Minerx117**
30/01/2020

### CODE REVIEW

**Review by aaronsace — COMMENTED** [review]
Awesome, I'll check it later and update the release.

File: MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1 [hunk 1]
```diff
@@ -27,18 +27,10 @@ if ($CUDAVersion -and [System.Version]$CUDAVersion -lt [System.Version]$Required
...
- }
-elseif ($CUDAVersion -lt [System.Version]"10.1.0") { 
-    $HashSHA256 = "8C9C1DFB454A8F56B76CBFB0E1071FAD03D908560CFF05E1DE462825E83DB1A4"
-    $Uri = "https://github.com/xmrig/xmrig-nvidia/releases/download/v2.14.5/xmrig-nvidia-2.14.5-cuda10-win64.zip"
- }
-else { 
-    $HashSHA256 = "9450E5483CBBDBEBDB706087E5B468FB716028751667C3F663E07395DCBC0476"
-    $Uri = "https://github.com/xmrig/xmrig-nvidia/releases/download/v2.14.5/xmrig-nvidia-2.14.5-cuda10_1-win64.zip"
+ 
+if {
```
Comment (aaronsace) [inline MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: Mistake?

Comment (Minerx117) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: not really your so far behide with xmrig it needs to be re written..

Comment (Minerx117) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: update your algos ie randomx randomsfx zergpool  im not going to do it for you

Comment (Minerx117) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: i guess so

Comment (Minerx117) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: cant be fucked to be honest

Comment (Minerx117) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: algos need updating

Comment (Minerx117) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: fixed the cuda warning Mistake. 
new xmrig is cuda 10.1 only xmrig.ps1 needs to be re written to fix --algo selection and api is not longer working. this is beyond my capabilities

Comment (aaronsace) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: Maybe leave this one out for now...? 

Can always tackle this binary on a separate pull request...

Are all the other files complete?

Comment (Minerx117) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: done, all complete

Comment (Minerx117) [reply MinersLegacy/XmrigCryptonight-v5.5.3-NVIDIA.ps1]: running through test rigs atm all good so far,
need to add Verushash algo support ZergPool CPU (most profitable atm)
i'll do it in a couple hours

**Review by aaronsace — APPROVED** [review]

**Pull request merged**